app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::CategoriesController < ApplicationController
+
+  def index
+    categories = Category.all
+    render json: { data: categories }, each_serializer: Categories::IndexSerializer
```

Comment:
Why do you use `data` as a key name? Why not use the resource name?

Reply:
We just changed the name to categories which is the right resource name.